@xen-orchestra/lite/src/App/PoolTab/PoolUpdates.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { Map } from 'immutable'
+import { withState } from 'reaclette'
+
+import IntlMessage from '../../components/IntlMessage'
+import Table, { Column } from '../../components/Table'
+import XapiConnection, { Host, ObjectsByType, PoolUpdate } from '../../libs/xapi'
+import { formatSize } from '../../helpers/format'
+
+const COLUMN: Column<PoolUpdate>[] = [
+  {
+    header: <IntlMessage id='name' />,
+    render: update => update.name,
+  },
+  {
+    header: <IntlMessage id='description' />,
+    render: update => update.description,
+  },
+  {
+    header: <IntlMessage id='version' />,
+    render: update => update.version,
+  },
+  {
+    header: <IntlMessage id='release' />,
+    render: update => update.release,
+  },
+  {
+    header: <IntlMessage id='size' />,
+    render: update => formatSize(update.size),
+  },
+]
+
+interface ParentState {
+  objectsByType: ObjectsByType
+  objectsFetched: boolean
+  xapi: XapiConnection
+}
+
+interface State {}
+
+interface Props {
+  poolId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {}
+
+interface Computed {
+  availableUpdates?: PoolUpdate[]
+  hosts?: Map<string, Host>
+}
+
+const PoolUpdates = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    computed: {
+      availableUpdates: async function (state) {
+        const promises: Promise<unknown>[] = []
+        state.hosts?.forEach(({ $ref }) =>
+          promises.push(state.xapi.call('host.call_plugin', $ref, 'updater.py', 'check_update', {}))
+        )
+
+        let poolUpdates: Map<string, PoolUpdate> = Map()
```

Comment:
I am using a `Map` for easily test if the update is already stored.
In case of one `pool` have multiple `hosts`, and two `hosts` have the same `update`, we don't want to show two times the same update in the `pool update` list.